sklearn/decomposition/_dict_learning.py

```diff
@@ -63,30 +64,30 @@ def _sparse_encode(X, dictionary, gram, cov=None, algorithm='lasso_lars',
         threshold: squashes to zero all coefficients less than regularization
         from the projection dictionary * data'
 
-    regularization : int | float
+    regularization : int | float, default=None
```

Comment:
I think we usually say `: int or float, default=None` in this case.

Reply:
What should be done with cases like line 230 where the default is None which means in fact 0.1 * n_features ? I have been inconsistent with those cases through the script. 

`n_nonzero_coefs` : int, 0.1 * n_features by default`